pkg/operator/ceph/config/monstore_test.go

```diff
@@ -72,6 +72,7 @@ func TestMonStore_Set(t *testing.T) {
 	e = monStore.Set("mon.*", "unknown_setting", "10")
 	assert.Error(t, e)
 	assert.Contains(t, execedCmd, " config set mon.* unknown_setting 10 ")
+	execInjectErr = false
```

Comment:
Why do you need to set this to false at the end of the test?

Reply:
Removed.